presto-main/src/main/java/com/facebook/presto/cost/FilterStatsCalculator.java

```diff
@@ -157,7 +157,7 @@ private PlanNodeStatsEstimate symbolToLiteralGreaterThan(Symbol symbol, double l
             double filterFactor = range.overlapPercentWith(intersectRange);
```

Comment:
You did not fix all the usages.
Also change corresponding code to match the name change.

E.g. `SymbolStatsAssertion`